CorsixTH/Lua/hospital.lua

```diff
@@ -1399,7 +1399,7 @@ function Hospital:receiveMoneyForTreatment(patient)
     else
       reason = _S.transactions.cure_colon .. " " .. casebook.disease.name
     end
-    local amount = self:getTreatmentPrice(disease_id)
+    local amount = pay_amount or 0
```

Comment:
If you receive 0, does it make sense to perform payment at all, and create a floating dollarsign `$0` in the code below?

Reply:
this got changed already, closing.